Content.Server/Voting/Managers/VoteManager.DefaultVotes.cs

```diff
@@ -158,7 +158,13 @@ private void CreatePresetVote(IPlayerSession? initiator)
 
         private void CreateMapVote(IPlayerSession? initiator)
         {
-            var maps = _gameMapManager.CurrentlyEligibleMaps().ToDictionary(map => map, map => map.MapName);
+            var maps = new Dictionary<string, GameMapPrototype>();
+            var eligibleMaps = _gameMapManager.CurrentlyEligibleMaps().ToList();
+            maps.Add("Secret", _random.Pick(eligibleMaps));
```

Comment:
This should be fluent I think

Reply:
I don’t think there is a check that the secret won and the check goes just along the line. So it's impossible to translate

Reply:
huh?

Reply:
The vote always shows the code name of the cards and it is not translated, why then translate the secret version? In any case, I have no idea how to solve this problem.

Reply:
it's fucking maps not cards smh.
You can do 
```cs
if (title == Loc.GetString("secret-map")) {
```

Reply:
The fact that your if statement is basically compares names is a much huger crutch. Ion see anything bad with comparing fluent strings. You didn't explain why you think so

Reply:
> The fact that your if statement is basically compares names is a much huger crutch. Ion see anything bad with comparing fluent strings. You didn't explain why you think so
In any case, I would like to see an option suggestion without the if (title = "Secret") check

Reply:
> The fact that your if statement is basically compares names is a much huger crutch. Ion see anything bad with comparing fluent strings. You didn't explain why you think so

In any case, I would like to see an option suggestion without the if (title = "Secret") check